tests/unittests/benchmark/task/test_quadratics.py

```diff
@@ -0,0 +1,22 @@
+""" Tests for the Quadratics Task. """
+
+from orion.benchmark.task.quadratics import QuadraticsTask
+from typing import ClassVar, Type
+
+
+class TestQuadraticsTask:
+    task: ClassVar[Type[QuadraticsTask]] = QuadraticsTask
+
+    def test_seeding(self):
+        seed = 123
+
+        task_a = self.task(max_trials=123, seed=seed)
+        task_b = self.task(max_trials=123, seed=seed)
+        assert task_a.a2 == task_b.a2
+        assert task_a.a1 == task_b.a1
+        assert task_a.a0 == task_b.a0
+
+        task_c = self.task(max_trials=123, seed=seed + 1)
+        assert task_b.a2 != task_c.a2
+        assert task_b.a1 != task_c.a1
+        assert task_b.a0 != task_c.a0
```

Comment:
`call` should be tested with and without context. It should verify that the `self.a`s are used instead of the ones passed.

By the way, I think the `a`s argument of `call` should be optional. If we set without context, the hparams will not contain the `a`s.

Reply:
Agreed.
The `a` arguments were made optional in 7181cad7a8679df197de50698a4cc3159d413bed.
I've added tests to check that the coefficients are ignored in ab55d278d4423a88e478f0ab47ee1fb04ca65511